.vscode/extensions.json

```diff
@@ -1,10 +1,11 @@
 {
     "recommendations": [
-      "ms-vscode.csharp",
+      "ms-dotnettools.csharp",
       "ms-vscode.cpptools",
       "ms-vscode.mono-debug",
-      "wghats.vscode-nxunit-test-adapter",
       "visualstudioexptteam.vscodeintellicode",
       "streetsidesoftware.code-spell-checker",
+      "wghats.vscode-nxunit-test-adapter",
+      "derivitec-ltd.vscode-dotnet-adapter",
```

Comment:
Are trailing commas allowed in `extensions.json`?

Reply:
It seems they are cos it works for me 🤷